cl/src/merkle.rs

```diff
@@ -67,15 +67,14 @@ pub fn verify_path(leaf: [u8; 32], path: &[PathNode], root: [u8; 32]) -> bool {
 }
 
 pub fn path<const N: usize>(leaves: [[u8; 32]; N], idx: usize) -> Vec<PathNode> {
```

Comment:
nit: what's the advantage in hardcoding the merkle tree length?

Reply:
mostly "because we can". We can drop later it if we find it is inconvenient for some reason.